packages/client/hmi-client/src/components/documents/DocumentCard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+	<Card>
+		<template #header>
+			<section class="content">
+				<div class="image"></div>
```

Comment:
Why did you not brought over the image function?

Reply:
What do you mean?

Reply:
I think they come from the features, no?